.travis.yml

```diff
@@ -42,8 +62,15 @@ install_linux: &install_linux
   - export OPAMYES=1
   - opam install sedlex xml-light extlib rope ptmap
   # Install neko
-  - git clone https://github.com/HaxeFoundation/neko.git ../neko
-  - pushd ../neko
+  - if [ ! -d "$HOME/neko" ]; then
+      mkdir $HOME/neko;
+      pushd $HOME/neko;
+      git clone https://github.com/HaxeFoundation/neko.git .;
+    else
+      pushd $HOME/neko;
+      git pull origin master || git clone https://github.com/HaxeFoundation/neko.git .;
```

Comment:
I think a `git fetch --all; git reset --hard origin/master` would be more reliable when there is merge conflict.